mak/SRCS

```diff
@@ -52,8 +52,19 @@ SRCS=\
 	src\core\sync\rwmutex.d \
```

Comment:
No. I removed it purposefully, because I was afraid that it would break something to just change the module name even though it was wrong. I just forgot to update the commit message. So, I'll fix that shortly. I'll figure out what to do about pthread_np in a different PR.